k2/csrc/util.cc

```diff
@@ -0,0 +1,38 @@
+// k2/csrc/util.cc
+
+// Copyright (c)  2020  Fangjun Kuang (csukuangfj@gmail.com)
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/csrc/util.h"
+
+#include <stdlib.h>
+
+#include "glog/logging.h"
+
+namespace k2 {
+
+void *MemAlign(size_t nbytes, size_t alignment) {
```

Comment:
I prefer name `MemAlign(ed)Malloc` as posix naming style is just like: we only do (padding) alignment..

Reply:
Done.